ui/src/chat/ChatMessage/DateDivider.tsx

```diff
@@ -1,17 +1,23 @@
 import React from 'react';
 import cn from 'classnames';
-import { makePrettyDay, pluralize } from '../../logic/utils';
+import { makePrettyDay, pluralize } from '@/logic/utils';
+import { useChatState } from '@/state/chat';
 
 interface DateDividerProps {
   date: Date;
   unreadCount?: number;
+  viewRef?: (node?: Element | null | undefined) => void;
 }
 
-export default function DateDivider({ date, unreadCount }: DateDividerProps) {
+export default function DateDivider({
+  date,
+  unreadCount,
+  viewRef,
```

Comment:
This really should use `React.forwardRef` instead

Reply:
the library *might* not play well with it – but i'll poke around and see

Reply:
Okay, just pushed.